### DISCUSSION TITLE
Is running wiki.js as a service the same as using nginx ?  Have an error... not sure how to fix.

### DISCUSSION THREAD

**Discussion by codejoy** [requarks/wiki#7349] [Help / Questions]
I am seeing a error pop up in wiki.js sparadoically , then saw a possible fix of editing the nginx.conf file, trouble is to the best of my knowledge I don't use nginx, unless it is being used somehow inside wiki.js when it is ran as a service ?  The error is:

Network error: JSON.parse: unexpected character at line 1 column 1 of the JSON data

Someone posted said error here and got an answer:

 https://github.com/requarks/wiki/discussions/5325

But again, that answer seemed only valid for nginx.conf files, which I don't see in my simple wiki.js installation, so not sure what else to look for.   I realized though wikim is running on a server as a service that might be reversed proxied though another server running apache

**Comment by vowqg** [comment]
maybe the post body is too large。
in the config.yml, 
`bodyParserLimit: 10mb`
